{
  "modified": "2024-06-25T13:40:40Z",
  "published": "2024-06-25T13:40:40Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5775",
  "aliases": [
    "SNYK-PYTHON-PYSQLILIBRARYV1-5932263"
  ],
  "summary": "Malicious code in pysqlilibraryv1 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pysqlilibraryv1",
        "purl": "pkg:pypi/pysqlilibraryv1"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PYSQLILIBRARYV1-5932263"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-04571",
        "import_time": "2024-06-28T02:50:15.90709745Z",
        "modified_time": "2024-06-25T13:40:40Z",
        "sha256": "22014327bc85cdfcdb14a98b5f4c178ab419860fa199bd525fc157639bc47ec5",
        "source": "reversing-labs",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
